Assets/Scripts/PlayerPositionHandler.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerPositionHandler : MonoBehaviour
+{
+    public GameObject xrOrigin;
+
+    public GameObject xrCamera;
+
+    public void ProcessKeyframe(KeyframeData keyframe)
+    {
+        if (!enabled) return;
+
+        var humanoidPosition = keyframe?.message.humanoidPosition;
+        if (humanoidPosition != null && humanoidPosition.Count == 3)
+        {
+            Vector3 newPosition = CoordinateConventionHelper.ToUnityVector(humanoidPosition);
+
+            Vector3 delta = newPosition - xrCamera.transform.position;
+            delta = new Vector3(delta.x, 0.0f, delta.z); // Ignore y-axis delta
```

Comment:
It's fine to ignore the y here. In the future, for multi-story houses, we'll have to find a way to adjust height, too (presumably Quest somehow exposes the headset's height above the physical ground).

Reply:
Indeed, great point.